pkg/blobinfocache/prioritize/prioritize.go

```diff
@@ -1,4 +1,6 @@
-package blobinfocache
+// Package prioritize provides utilities for prioritizing locations in
+// types.BlobInfoCache.CandidateLocations.
+package prioritize
```

Comment:
I think no external user needs to care; move this into a `pkg/blobinfocache/internal` (or `pkg/blobinfocache/internal/prioritize`? I don’t have a opinion really).

Reply:
> I think no external user needs to care; move this into a `pkg/blobinfocache/internal` (or `pkg/blobinfocache/internal/prioritize`? I don’t have a opinion really).

Part of me thinks "external users might want access to these utilities when they write their own `BlobInfoCache` implementation" and part of me things "nobody else is ever going to write a `BlobInfoCache` implementation" ;).  So I don't have a strong opinion either; moved under `internal/` with 15dfbbc -> ca54d11 (which also rebases onto master).

Reply:
> Part of me thinks "external users might want access to these utilities when they write their own `BlobInfoCache` implementation" and part of me things "nobody else is ever going to write a `BlobInfoCache` implementation" ;)

Yes, both.  If anyone felt a strong need to write a different cache implementation, that would primarily be a reason improve the default one.